lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -44,6 +46,7 @@ def refresh_course(self):
         """
         Re-fetch the course from the database so that the object being dealt with has everything added to it.
         """
+        # pylint: disable=attribute-defined-outside-init
```

Comment:
Nit: As a test class, this violation is generally acceptable.  Consider just adding the pragma to the top of the file, instead of interspersed throughout.